react_app/src/reducers/index.js

```diff
@@ -5,10 +5,10 @@ let default_data = {}
 
```

Comment:
did you make changes to this file?

Reply:
It doesn't seem like it. I think I must've applied the formatter on my vscode on it.